test/src/main/java/org/apache/accumulo/harness/SharedMiniClusterBase.java

```diff
@@ -141,14 +140,14 @@ public static synchronized void startMiniClusterWithConfig(
   }
 
   private static String getTestClassName() {
-    Predicate<Class<?>> findITClass = c -> c.getSimpleName().endsWith("IT");
+    Predicate<Class<?>> findITClass =
+        c -> c.getSimpleName().endsWith("IT") || c.getSimpleName().endsWith("SimpleSuite");
     Function<Stream<StackFrame>,Optional<? extends Class<?>>> findCallerITClass =
         frames -> frames.map(StackFrame::getDeclaringClass).filter(findITClass).findFirst();
     Optional<String> callerClassName =
         StackWalker.getInstance(RETAIN_CLASS_REFERENCE).walk(findCallerITClass).map(Class::getName);
     // use the calling class name, or default to a unique name if IT class can't be found
-    return callerClassName.orElse(String.format("UnknownITClass-%d-%d", System.currentTimeMillis(),
-        RANDOM.get().nextInt(Short.MAX_VALUE)));
+    return callerClassName.orElse("UnknownITClass");
```

Comment:
Could probably just do `.orElseThrow()` here, since this really shouldn't happen.